src/hooks/aelin/useAelinTokenList.tsx

```diff
@@ -23,18 +23,23 @@ const getTokenList = (chainId: ChainsValues) => {
 const useTokenListQuery = (appChainId: ChainsValues) => {
   return useSWR(['token-list', appChainId], async () => {
     const response: TokenListResponse = await getTokenList(appChainId)
+    let tokens: Token[] = response.tokens
 
     if (
       appChainId === Chains.optimism ||
       appChainId === Chains.arbitrum ||
       appChainId === Chains.polygon
     ) {
-      return response.tokens.filter(
-        ({ chainId }: { chainId: number }) => Number(chainId) === appChainId,
-      )
+      tokens = tokens.filter(({ chainId }: { chainId: number }) => Number(chainId) === appChainId)
     }
 
-    return response.tokens
+    if (appChainId === Chains.polygon) {
+      tokens = tokens.filter(({ symbol }) => symbol.toLowerCase() !== 'matic')
+    } else {
+      tokens = tokens.filter(({ symbol }) => symbol.toLowerCase() !== 'eth')
```

Comment:
Is there a way to filter by token address instead of token symbol? I'm not sure if it's possible, but it would be good if it is.

Reply:
Yeah, it's possible, but the main point was to exclude tokens that mimic to native currency

Reply:
Ahh, I got it!